packages/airnode-admin/test/e2e/cli.feature.ts

```diff
@@ -48,9 +48,9 @@ describe('CLI', () => {
     if (DEBUG_COMMANDS) console.log(`Executing command: ${formattedCommand}`);
     try {
       return execSync(`node ${CLI_EXECUTABLE} ${formattedCommand}`).toString().trim();
-    } catch (e: any) {
+    } catch (e) {
       // rethrow the output of the CLI
-      throw new Error(e.stdout.toString().trim());
+      throw new Error((e as any).stdout.toString().trim());
```

Comment:
Why is this change needed? If you don't state it explicitly the variable is implicitly of type `any`. There's also no need for type assertion.

Reply:
Not sure why is this included - I think I was debugging some stuff and forget to revert this.